src/utils/helpers.ts

```diff
@@ -217,3 +217,6 @@ export function isArray(value: unknown): value is any[] {
 export function isBoolean(value: unknown): value is boolean {
   return typeof value === 'boolean';
 }
+
+// get location hash without #
+export const getLocationHash = () => decodeURI(location.hash).substr(1);
```

Comment:
There may be problems with server side rendering here (`location`). I think it needs to be wrapped in a try catch.